pkg/controller/catalogsourceconfig/catalogsourceconfig_controller.go

```diff
@@ -81,19 +81,11 @@ type ReconcileCatalogSourceConfig struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileCatalogSourceConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	log.Printf("Reconciling CatalogSourceConfig %s/%s\n", request.Namespace, request.Name)
-	// Reconcile kicked off, message Sync Channel
-	status.SendSyncMessage(nil)
-
-	// If err is not nil at the end of this function message the syncCh channel
-	var err error
-	defer func() {
-		if err != nil {
-			status.SendSyncMessage(err)
-		}
-	}()
+	operatorstatus.SendEventMessage(nil)
+
 	// Fetch the CatalogSourceConfig instance
 	instance := &v2.CatalogSourceConfig{}
-	err = r.client.Get(context.TODO(), request.NamespacedName, instance)
+	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
 	if err != nil {
 		if errors.IsNotFound(err) {
```

Comment:
What about if the get fails, and it's not from an `IsNotFound` error? Don't we want to increment the value in cluster status then?

Reply:
Same comment about both controllers btw

Reply:
I believe this is a good point. I will update the code to increment the number of failed transitions when an error other than `IsNotFound` is encountered during a `get` request in all places.